### PR TITLE
apps/examples: Modify debug logs for smartfs_powercut test app

### PR DESCRIPTION
**Pull request by amogh-samsung**
- Add debug macro for print statements in smartfs_powercut app.

Signed-off-by: Amogh Hassija <a.hassija@samsung.com>

### CODE REVIEW

File: apps/examples/smartfs_powercut/smartfs_powercut_main.c [hunk 1]
```diff
@@ -699,4 +707,4 @@ int smartfs_powercut_main(int argc, char *argv[])
 	}
 	
 	return OK;
-}
+
```
Comment (amandeepc) [inline apps/examples/smartfs_powercut/smartfs_powercut_main.c:710]: Need the closing bracket. Change needs to be undone.

Comment (amogh-samsung) [reply apps/examples/smartfs_powercut/smartfs_powercut_main.c]: My mistake, updated, thanks.

**Review by jeongchanKim — COMMENTED** [review]
This is an app, so I'm not sure this debug macro is needed or not. I think printf is enough for failure case.
Do you have any reason to use debug instead of printf?

File: apps/examples/smartfs_powercut/smartfs_powercut_main.c [hunk 2]
```diff
@@ -64,10 +64,18 @@
 #include <crc16.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
+#include <debug.h>
 
 #include <tinyara/fs/ioctl.h>
 #include <tinyara/config.h>
 
+/* Debug statement definition for smartfs powercut test app */
+#ifdef CONFIG_DEBUG_FS_ERROR
```
Comment (Taejun-Kwon) [inline apps/examples/smartfs_powercut/smartfs_powercut_main.c:73]: It is application, so this is not good choice.
I think we'd better define debug level here.
please see below.
```
#define DEBUG_NONE 0
#define DEBUG_LOW   1
#define DEBUG_HIGH  2

#define DEBUG_LEVEL DEBUG_LOW //we can change this in code level.
#if (DEBUG_LEVEL == DEBUG_LOW)
#define PCDBG(format, ...)   printf(EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
#define PCVDBG(format, ...)
#elif (DEBUG_LEVEL == DEBUG_HIGH)
#define PCDBG(format, ...)   printf(EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
#define PCVDBG(format, ...) printf(EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
#else // NONE
#define PCDBG(format, ...)
#define PCVDBG(format, ...)
#endif
```

File: apps/examples/smartfs_powercut/smartfs_powercut_main.c [hunk 3]
```diff
@@ -64,10 +64,18 @@
...
+#define appdbg(format, ...) dbg(format, ##__VA_ARGS__)
+#else
+#define appdbg(...)
+#endif
+
```
Comment (deepaksrma) [inline apps/examples/smartfs_powercut/smartfs_powercut_main.c:78]: can you remove this line?

**Review by Taejun-Kwon — APPROVED** [review]

**Comment by amogh-samsung** [comment]
Abandoned due to lack of priority.
Might be taken up again later.
NOTE: - Latest commits added to this PR DO NOT belong here

**Pull request closed without merging**